Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerSystemComponent.cpp

```diff
@@ -19,7 +19,7 @@
 
 namespace AWSGameLift
 {
-    AZ_CVAR(bool, sv_useGameLiftServer, false, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Activate GameLift server manager and SDK");
+    AZ_CVAR(bool, sv_gameLiftEnabled, false, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Activate GameLift server manager and SDK");

```

Comment:
is there a corresponding o3de.org docs update to reflect this change?

Reply:
Now there is... thanks!
https://github.com/o3de/o3de.org/pull/2357